docs/dotprompt.md

```diff
@@ -42,12 +42,12 @@ import { prompt } from '@genkit-ai/dotprompt';
 Then, load the prompt using `prompt('file_name')`:
 
 ```ts
-const greetingPrompt = await prompt('greeting');
+const menuPrompt = await prompt('menu');
```

Comment:
We're calling this `menuPrompt` here, but it looks like the prompt above is still providing a greeting in the style of...